package.json

```
@@ -42,7 +42,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.paypal.com/paypal/paypal-auth.git"
+    "url": "https://github.com/paypal/paypal-identity-components.git"
```

Thanks for making this change! Can you update the url to use `git` instead of `https`?
```suggestion
    "url": "git://github.com/paypal/paypal-identity-components.git"
```